src/routes/app/EditProfilePage.tsx

```diff
@@ -337,7 +337,12 @@ import Footer from "../../components/Footer";
                     </div>
 
                     <div className="editprofile-form-input" >
-                        <label htmlFor="editprofile-radius" >Radius</label>
+                        <label htmlFor="editprofile-radius" >Radius
+                            <span className="tooltip">
+                                <i className="fa-solid fa-circle-info"></i>
+                                <span className="tooltiptext">The default search area for exploring nearby rides. ChainLink will default to this radius when display upcoming rides on your explore page.</span>
```

Comment:
Doesn't read quite right, you put "when display" instead of "when displaying":
![image](https://github.com/user-attachments/assets/951e97b0-21c0-4f9a-8fe2-0d93fe99309b)

The ticket had this copy: "This defines the default search area for exploring nearby rides. Chainlink will automatically display upcoming rides within this radius on your explore page."

Reply:
Updated